### PR TITLE
Improve env instructions

### PR DESCRIPTION
**Pull request by alexpott**
I've now got this working with both selenium-server and chromedriver. This PR attempts to improve the docs in the .env.example to make it easier.

### CODE REVIEW

File: core/tests/Drupal/Nightwatch/nightwatch.conf.js [hunk 1]
```diff
@@ -10,9 +10,9 @@ module.exports = {
   },
   test_settings: {
     default: {
-      selenium_port: 9515,
+      selenium_port: process.env.WEBDRIVER_PORT,
```
Comment (dawehner) [inline core/tests/Drupal/Nightwatch/nightwatch.conf.js]: can we add this to the .env file?

File: core/.env.example [hunk 2]
```diff
@@ -1,11 +1,37 @@
...
 CHROME_STANDALONE=false
-NIGHTWATCH_OUTPUT=reports/nightwatch
+
+# The URL where webdriver is available. The default values work well with
+# chromedriver. In order to use selenium-driver, set CHROME_STANDALONE to true
+# and remove the comment before WEBDRIVER_PATH_PREFIX and change WEBDRIVER_PORT
+# to the correct port, often 4444.
 WEBDRIVER_HOSTNAME=localhost
-#WEBSERVER_USER=
+WEBDRIVER_PORT=9515
```
Comment (alexpott) [inline core/.env.example]: @dawehner it's here. I tried to group related settings.

Comment (dawehner) [reply core/.env.example]: Oops, sorry, you are right

File: core/tests/Drupal/Nightwatch/nightwatch.conf.js [hunk 3]
```diff
@@ -10,9 +10,9 @@ module.exports = {
...
       selenium_host: process.env.WEBDRIVER_HOSTNAME,
-      default_path_prefix: '',
+      default_path_prefix: process.env.WEBDRIVER_PATH_PREFIX ? process.env.WEBDRIVER_PATH_PREFIX : '',
```
Comment (dawehner) [inline core/tests/Drupal/Nightwatch/nightwatch.conf.js]: you can use ```proces.env.WEBDRIVER_PATH_PREFIX || ''``` instead

File: core/tests/Drupal/Nightwatch/nightwatch.conf.js [hunk 4]
```diff
@@ -12,7 +12,7 @@ module.exports = {
     default: {
       selenium_port: process.env.WEBDRIVER_PORT,
       selenium_host: process.env.WEBDRIVER_HOSTNAME,
-      default_path_prefix: proces.env.WEBDRIVER_PATH_PREFIX || '',
```
Comment (dawehner) [inline core/tests/Drupal/Nightwatch/nightwatch.conf.js]: How did you managed to make a typo in a c&p :P

**Pull request merged**